src/gui/optionsdlg.h

```diff
@@ -60,6 +60,8 @@ namespace Ui
 class OptionsDialog: public QDialog
 {
     Q_OBJECT
+    using ThisType = OptionsDialog;
```

Comment:
>I added this because ThisType is shorter and cleaner than OptionsDialog and ...

Not sure you get me, I was suggesting change `ThisType` to `ThisClass`, since this looks like creating a shorter alias for the class name.